### DISCUSSION TITLE
Does the Inspector work with React Native?

### DISCUSSION THREAD

**Discussion by ninjz** [statelyai/xstate#4960] [Q&A] [answered]
I'm trying to use the Stately Inspector with `createBrowserInspector` however I keep getting an error `Property 'Event' doesn't exist`. Is this tool not compatible with React Native?

**Comment by davidkpiano** [comment, marked as answer]
You may be interested in this: https://github.com/cpetzel/expo-xstate-inspect/tree/main

**Reply by ninjz** [reply]
Just what I was looking for. Thanks!
